cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts

```diff
@@ -12,4 +12,8 @@ export default abstract class BedspaceEditablePage extends Page {
 
     this.clickSubmit()
   }
+
+  getCharacteristics(alias: string): void {
```

Comment:
I think I'd rather pass in the alias - keep the alias name knowledge in one place (in the step definitions, rather than both the step definitions and the page helper class). But I think changing "get" to "assign" is a good call. How would you feel about just `assignCharacteristics(alias: string)` and the "ToAlias" is implicit by context/convention?

Reply:
yep sure, that makes sense :)